mediaQueries/mobile.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./mobileFirstStyles.css">
+</head>
+
+<body>
+    <header>
+        <img src="./rad32.png" class="headerimg" alt="headerimg">
```

Comment:
adding an image via css is usually done when you want that image to be the background to an element. 
adding an image via the `<img>` tag is done when you want that image to be a part of the flow of the page.
you did it the right way!